.travis.yml

```diff
@@ -1,63 +1,41 @@
-# This travis image builds roboteam_suite and runs the target 'world_tests'.
+# This travis image builds roboteam_suite and runs the target 'utils_tests'.
 # All other repositories of roboteam_suite are set to the master branch.
 # We need all dependencies to parse the cmake files. This can be optimized later,
 # but with caching this is not a large performance hit anyway.
 
+# Language C++, Ubuntu 18
+os: linux
+dist: bionic
 language: cpp
-os: osx
-osx_image: xcode11.2
-
-addons:
-  homebrew:
-    packages:
-      - zmq       # proto dependency
-      - zmqpp     # proto dependency
-      - protobuf  # proto dependency
-      - armadillo # world dependency
-      - ccache    # speeding up build
-      - ninja     # speeding up build
-      - qt        # roboteam_ai dependency
 
+# Services require docker, docker already contains everything
+services:
+  - docker
+
+# Disable git cloning cause we do that in docker
+git:
+  clone: false
+
+# This happens before your build
+before_install:
+  # Pull the docker image, not cached, caching is slower
+  - docker pull yuhanuncitgez/roboteamtwente:latest
```

Comment:
Does this not call your fork instead of ours? Or is this docker magic I don't know of ?

Reply:
docker pull pulls from

hub.docker.com, not from git, we need to set up a hub account for roboteam.